index.html

```diff
@@ -45,5 +47,246 @@ <h1 class="presentation"> I'm Jorge E. Prada M.<br>
+                    <li><img src="img/circle.png" alt="circle image"></li>
+                    <li><a class="text-decoration">2015</a></li>
+                </ul>
+            </div>
+            <div class="work1-description">
+                <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+            </div>
+            <div class="work-description-detail">
+                <ul>
+                    <li><a class="text-decoration blue-color">HTML</a></li>
+                    <li><a class="text-decoration blue-color">CSS</a></li>
+                    <li><a class="text-decoration blue-color">JavaScript</a></li>
+                </ul>
+            </div>
+            <div class="btn-see-project">
+                <a class="text-decoration blue-color">
+                    <button class="btn">See project</button>
+                </a>
+            </div>
+        </div>
+        <div class="work4">
+
+            <div>
+                <a>
+                    <img src="img/work4.png" alt="work image">
+                </a>
+            </div>
+            <div class="work1-title">
+                <h2>Multi-Post <br>
+                    Stories<br>
+                </h2>
+            </div>
+            <div class="work1-detail">
+                <ul>
+                    <li><a class="text-decoration">CANOPY</a></li>
+                    <li><img src="img/circle.png" alt="circle image"></li>
+                    <li><a class="text-decoration">Back End Dev</a></li>
+                    <li><img src="img/circle.png" alt="circle image"></li>
+                    <li><a class="text-decoration">2015</a></li>
+                </ul>
+            </div>
+            <div class="work1-description">
+                <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+            </div>
+            <div class="work-description-detail">
+                <ul>
+                    <li><a class="text-decoration blue-color">HTML</a></li>
+                    <li><a class="text-decoration blue-color">CSS</a></li>
+                    <li><a class="text-decoration blue-color">JavaScript</a></li>
+                </ul>
+            </div>
+            <div class="btn-see-project">
+                <a class="text-decoration blue-color">
+                    <button class="btn">See project</button>
+                </a>
+            </div>
+        </div>
+        <section class="about-myself">
+            <div class="about-myself-container">
+                <div class="title">
+                    <h2>About Myself</h2>
+                </div>
+                <div class="description">
+                    Hello I'm a software developer!. I can help you build a product, feature or website. Look through
+                    some
+                    of my work and experience!. If you like what you see and have a project you need coded, don't
```

Comment:
- [ ] Please `the about myself` section should have the margins (top, bottom, left, right) defined in the design.
- There is a weird distance around the about myself section,
- the distance between the text and the let's connect button doesn't match the design.

Your design 
![image](https://user-images.githubusercontent.com/74744787/229313758-17e45045-664f-4356-8ded-3c65c886a41c.png)

Figma 
![image](https://user-images.githubusercontent.com/74744787/229313773-892c9640-bca8-42f1-8967-207e35a386f6.png)

- [ ] Please `about myself` section should use the font and colors (background and font) defined for the template. 
- The title and the text colors don't match the design.
Your design 
![image](https://user-images.githubusercontent.com/74744787/229313921-65e76390-727a-4703-8b53-f7501f993dca.png)

Figma 
![image](https://user-images.githubusercontent.com/74744787/229313910-5bdcd986-4bf8-47cc-9275-b7dcee0eb1f9.png)

- [ Optional ]  The order of the icons doesn't match the design.
Your design
![image](https://user-images.githubusercontent.com/74744787/229314107-4bb46757-7833-4129-a115-b91ba0528082.png)

Figma 
![image](https://user-images.githubusercontent.com/74744787/229314121-0fa40089-5b80-44d5-a8a7-efde6913be7d.png)